data/genesis/demo-pos.toml

```diff
@@ -21,7 +23,7 @@ number = 0
 
 [[upgrade]]
 version = "0.3"
-start_proposing_view = 200
+start_proposing_view = 150
 stop_proposing_view = 1000
```

Comment:
lets decrease the `stop_proposing_view` window to something like 300 maybe?

Reply:
The test actually only waits for 200 blocks for the upgrade to happen and will fail otherwise. Not sure what setting it lower in this file accomplishes though.

Reply:
doesn't accomplish anything, its just weird to have such a large window for proposing